aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -111,13 +121,30 @@ Here's a sample `aspNetCore` element that configures `stdout` logging. The `stdo
     stdoutLogFile="\\?\%home%\LogFiles\stdout">
 </aspNetCore>
 ```
-See [Configuration via web.config](#configuration-via-webconfig) for an example of the `aspNetCore` element in the *web.config* file.
+
+See [Configuration with web.config](#configuration-with-webconfig) for an example of the `aspNetCore` element in the *web.config* file.
 
 ## ASP.NET Core Module with an IIS Shared Configuration
 
-The ASP.NET Core Module installer runs with the privileges of the **SYSTEM** account. Because the local system account doesn't have modify permission for the share path which is used by the IIS Shared Configuration, the installer will hit an access denied error when attempting to configure the module settings in *applicationHost.config* on the share.
+The ASP.NET Core Module installer runs with the privileges of the **SYSTEM** account. Because the local system account doesn't have modify permission for the share path which is used by the IIS Shared Configuration, the installer hits an access denied error when attempting to configure the module settings in *applicationHost.config* on the share.
+
+Unsupported workaround:
```

Comment:
This isn't unsupported, it's a perfectly valid scenario to install ANCM and then later enable shared config.

Reply:
This goes back to your remark here :point_right: https://github.com/aspnet/Docs/issues/1816#issue-173086212

You approved the language here :point_right: https://github.com/aspnet/Docs/issues/1816#issuecomment-242797281

I'll remove the word "unsupported" on the next commit. @Rick-Anderson note this in case the support folks come for my *head* later! 🔪 :boy: :smile: